_posts/2021-09-26-boa-release-13.md

```diff
@@ -0,0 +1,44 @@
+
+We currently support part of the language. In this release, our conformance has grown to 41.97% of the official ECMAScript Test Suite (Test262). In this release, we have closed ?? issues and merged ?? pull requests. You can check the full list of changes [here](https://github.com/boa-dev/boa/blob/v0.13/CHANGELOG.md).
+
+## New Features
+
+This release brings some new features, such as support for the [Relative Indexing Method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) (better known as `at()`). Thanks to @neeldug for adding this. Other additions include `replaceAll()`, `Object.values()`, `Object.preventExtensions()`, `splice()`, `sort()`, spreading objects and more.
+
+## Test 262
+
+Test262 is the implementation conformance test suite maintained by TC39. It's used by nearly all engines to measure how conformant they are to the specification. Boa pulls the tests in-tree and runs them against all PRs. You can find more about Test262 [here](https://github.com/tc39/test262).
+
+Since v0.11 we have 10% more conformance and reduced panics. To put that into perspective that's almost 10,000 more tests that pass since when version 0.11 was released. We've dropped from 26 tests panicing to zero.
+
+[potential image of conformance chart?]
+
+## Keeping up with Rust!
+
+As we work on Boa Rust is changing underneath us. This means we need to make sure our code is conforming the latest standards and we're taking advantage of the best optimizations Rust can provide. In order to do this we rely on Clippy, this is Rust's in-house linter. We updated our code to [respect the latest Rust version](https://github.com/boa-dev/boa/pull/1352) and update [formatting](https://github.com/boa-dev/boa/pull/1356). Often a nice bonus of doing this is getting performance improvements for free as Clippy gets more "smarter". Thanks to @neeldug and @RageKnify for the work in this area.
+
+## Keeping things in order
+
+Previously object properties were stored using [`FXHash`](https://crates.io/crates/fxhash). Despite being very fast it was not guarunteeing any order. Properties need to retain the same order they were entered. In order to achieve this we switched over to [IndexMap](https://docs.rs/indexmap/1.7.0/indexmap/). Traits in Rust make this easy as IndexMap was designed to be a drop-in replacement for other HashMap implementations by following the same Trait. Performance losses were minimal on some benchmarks and we actually made some gains in others. Thanks to @raskad their work on this
+
+## Re-design VM for better space efficiency
+
+There is still on-going work to utilise a VM, this is happening [alongside](https://github.com/boa-dev/boa/blob/master/docs/vm.md#state-of-play) how Boa runs today. Although its not exposed yet there has been plenty of refactorings to the VM in order to make it performant. https://github.com/boa-dev/boa/pull/1361
+
```

Comment:
As @Razican suggested in https://github.com/boa-dev/boa/pull/1590 this would be my idea for a section on closure functions (Ignore the backslash escape at the end of the rust codeblock. I could not figure out how to escape the triple backticks inside a suggestion block):

```suggestion

## Calling rust closures from javascript

In addition to registering rust functions as global javascript functions, our api has been expanded to register rust closures. This allows to capture variables in closures that can be called from javascript:
```rust
let mut context = Context::new();

let add_value = 1.0;

context.register_global_closure("plusOne", 1, move |_, args, _| {
    let argument = args.get(0).unwrap().as_number().unwrap();

    Ok(JsValue::new(argument + add_value))
})?;

assert_eq!(context.eval("plusOne(41)")?, 42.into());
\```
To see the full capabilities of this feature, take a look at our [examples](https://github.com/boa-dev/boa/blob/master/boa/examples/closures.rs). Thanks to @HalidOdat and @jedel1043 for their work on this.

```

Reply:
You can use more than three ` to delimit a suggestion block. Edited the suggestion 😊

Reply:
I like it!, ive put it in